packages/amplify_datastore/example/ios/unit_tests/DataStorePluginUnitTests.swift

```diff
@@ -37,7 +37,7 @@ var eventSentExp: XCTestExpectation?
 class DataStorePluginUnitTests: XCTestCase {
 
     var pluginUnderTest: SwiftAmplifyDataStorePlugin = SwiftAmplifyDataStorePlugin()
-    var flutterModelSchemaRegistration: FlutterModels = FlutterModels()
+    var flutterModelSchemaRegistration: FlutterSchemas = FlutterSchemas()
```

Comment:
Please ignore this line, this changes has been corrected in the following commit.